zio-config/src/main/scala/zio/config/ConfigDescriptor.scala

```diff
@@ -57,11 +57,10 @@ object ConfigDescriptor {
   final case class Or[A, B](left: ConfigDescriptor[A], right: ConfigDescriptor[B])
       extends ConfigDescriptor[Either[A, B]]
 
-  def succeed[A](a: A): ConfigDescriptor[A] =
-    ConfigDescriptor.Succeed(a)
+  def empty[A]: ConfigDescriptor[Option[A]] = ConfigDescriptor.Empty()
 
   def sequence[A](configList: List[ConfigDescriptor[A]]): ConfigDescriptor[List[A]] =
-    configList.foldLeft(Succeed(Nil): ConfigDescriptor[List[A]])(
+    configList.foldLeft(Empty[List[A]]().xmap(_.toList.flatten)(_.headOption.map(List(_))))(
```

Comment:
Is it same as 
“Empty[A]().xmap(_.toList)(_.headOption)?”

Reply:
May be I am wrong. But that should give a Config[List[A]] in return?

Reply:
this will return `List[Option[Config]]` this is why I did `flatten`

Reply:
I will use `fold` to make it clearer